loading/lazyload/remove-element-and-scroll.tentative.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+  <title>Images with loading='lazy' load being removed and then scrolled to.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="common.js"></script>
+</head>
+
+<!--
+Marked as tentative until https://github.com/whatwg/html/pull/3752 is landed.
+-->
+
+<body>
+  <img id="in_viewport" src='resources/image.png'>
+  <div style="height:10000px;"></div>
+  <img id="below_viewport" src='resources/image.png' loading="lazy">
+
+  <script>
+    var in_viewport_element = document.getElementById("in_viewport");
+    var below_viewport_element = document.getElementById("below_viewport");
+
+    async_test(function(t) {
+      below_viewport_element.remove();
+      assert_false(is_image_fully_loaded(below_viewport_element));
+      in_viewport_element.addEventListener("load", t.step_func(function() {
+        window.scrollTo(0, 10000);
+        t.step_timeout(function() { assert_false(is_image_fully_loaded(below_viewport_element)); t.done(); }, 1000);
```

Comment:
My feeling is that even if the `below_viewport` image is not `loading=lazy`, this may still pass because it simply may not have finished loading entirely by the time the first's load event is called, right? (I could be missing something)

Reply:
You think 1 second is not enough? Should I try a bigger timeout, or can you think of some better way to test this?

Reply:
Sorry, I misunderstood this at first :/. Regardless, from https://web-platform-tests.org/writing-tests/testharness-api.html#timeouts-in-tests, the timeout may need to be longer anyways.

> Note that timeouts generally need to be a few seconds long in order to produce stable results in all test environments.

I do wonder if there is a way to write this without relying on a timeout though. I've asked around on #whatwg, but we'll see..

Reply:
Update: sounds like timeouts are going to be the way to go. Probably I'd make the timeout longer as per the recommendation